### PR TITLE
Bug 1677699: Provide further explanations on forcepull

### PR DESCRIPTION
**Pull request by codyhoag**
[BZ-1677699](https://bugzilla.redhat.com/show_bug.cgi?id=1677699)

@bmcelvee

### CODE REVIEW

File: dev_guide/managing_images.adoc [hunk 1]
```diff
@@ -483,6 +483,15 @@ the container. There are three possible values for `*imagePullPolicy*`:
 - `*IfNotPresent*` - only pull the image if it does not already exist on the node.
 - `*Never*` - never pull the image.
 
+[CAUTION]
+====
+You can xref:../install_config/build_defaults_overrides.adoc#manually-setting-global-build-overrides[enable the `forcePull` flag]
+in your global build configuration to force an image refresh from the registry
+every time a build starts. This forces an image access check on every build,
+removing the possibility of users accessing the build node's local cache of an
+image they don't have permission to access.
```
Comment (bmcelvee) [inline dev_guide/managing_images.adoc]: s/don't/do not

**Comment by bmcelvee** [comment]
One nit, otherwise LGTM :star2:

**Comment by codyhoag** [comment]
@adambkaplan could you SME review this change for accuracy? Thanks!

**Review by adambkaplan — APPROVED** [review]
/lgtm

Note that this only applies to OCP 3.x. OCP 4 is completely different.

**Comment by adambkaplan** [comment]
/retitle Bug 1677699: Provide further explanations on forcepull

**Comment by codyhoag** [comment]
@wewang58 can you provide a QE review for this change? Thanks!

**Comment by vikram-redhat** [comment]
@wewang58 bump

@xltian

**Comment by wewang58** [comment]
Docs LGTM, Thanks.

**Comment by codyhoag** [comment]
Thanks, @wewang58!

@bmcelvee do you mind merging/cherry picking this? Thanks!

**Comment by bmcelvee** [comment]
@codyhoag not at all!

**Pull request merged**